app/src/main/res/drawable/background_gradient.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
+    <item android:bottom="@dimen/activity_main_gradient_background_bottom_size">
+        <shape android:shape="rectangle">
+            <gradient
+                android:endColor="@color/honey_background_color"
```

Comment:
why are you using the same color in the gradient

Reply:
After testing different gradients and not finding a good combination with the colors I selected, I forgot to switch to a solid instead.

Reply:
if you will not use the gradient I think that all this file is unnecessary